qemu/qemu_test.go

```diff
@@ -89,22 +89,23 @@ func TestAppendEmptyMachine(t *testing.T) {
 	testAppend(machine, "", t)
 }
 
-var deviceNVDIMMString = "-device nvdimm,id=nv0,memdev=mem0 -object memory-backend-file,id=mem0,mem-path=/root,size=65536"
+var deviceNVDIMMString = "-device nvdimm,disable-modern=true,id=nv0,memdev=mem0 -object memory-backend-file,id=mem0,mem-path=/root,size=65536"
```

Comment:
@sboeuf You do not need disable-modern for NMDIMM. We need it only for virt-io pci devices. It may not even work